### ISSUE TITLE
Incompatible with Laravel Jetstream auth middleware?

### ISSUE THREAD

**Issue by tanthammar** [barryvdh/laravel-translation-manager#385]
Unable to access the `translations` route if the middleware in config isset to `auth` in a fresh Laravel **Jetstream** project.
If I set the middleware to `web` I can access the route.

* Laravel 8.24.0
* Laravel Jetstream 2.1
* laravel translation manager 0.5.9

## Steps to reproduce:
```
laravel new testproject --jet
//Choose [0]Livewire

cd testproject
touch database/database.sqlite
composer install
npm install && npm run dev
```
Change `.env` database connection to `sqlite`

```
composer require barryvdh/laravel-translation-manager
php artisan vendor:publish --provider="Barryvdh\TranslationManager\ManagerServiceProvider" --tag=migrations
php artisan migrate
php artisan vendor:publish --provider="Barryvdh\TranslationManager\ManagerServiceProvider" --tag=config
```

(I'm using Laravel Valet)

* create a user
* login to the site
* try accessing the `/translations` url
* **error:** you will be redirected to the `/dashboard` url

If I change the config to use the `web` middleware I can access the `/translations` url.

**Comment by PierreMatos** [comment]
yep, having the same issue.

**Comment by renepardon** [comment]
I'm also using this package with Laravel Jetstream and dont have problems. See https://github.com/barryvdh/laravel-translation-manager/issues/386

**Comment by tanthammar** [comment]
Thank you @renepardon I actually misread the documentation. In my mind I didn't read properly, I just assumed that it would be for 5.2 or earlier versions. 
Agree with you, that the initial config should be for later versions of Laravel. :)
Solved by #386
